Rssguard fails to build on armv7hl and aarch64 due to mimesis

### Brief description of the issue.
Rssguard fails to build on armv7hl and aarch64.

#### How to reproduce the bug?
On  armv7hl and aarch64

```
1. mkdir build && cd build
2. lrelease-qt5 ../build.pro
3. %{qmake_qt5} ../build.pro -r PREFIX=%{_prefix} LIB_INSTALL_DIR=%{_lib } CONFIG+=ltcg
4. %make_build. 

```
#### What was the expected result?

Build successful on all targeted arches.

#### What actually happened?

Build fails on aarch64 and armv7hl. It's specifically the mimesis compilation that fails with:

```
g++ -c -pipe -std=c++17 -flto=auto -ffat-lto-objects -fexceptions -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -mbranch-protection=standard -fasynchronous-unwind-tables -fstack-clash-protection -O3 -std=gnu++1z -flto -fno-fat-lto-objects -Wall -Wextra -D_REENTRANT -fPIC -DAPP_VERSION="\"3.7.1\"" -DAPP_NAME="\"RSS Guard\"" -DAPP_LOW_NAME="\"rssguard\"" -DAPP_LOW_H_NAME="\".rssguard\"" -DAPP_LONG_NAME="\"RSS Guard 3.7.1\"" -DAPP_AUTHOR="\"Martin Rotter\"" -DAPP_EMAIL="\"rotter.martinos@gmail.com\"" -DAPP_URL="\"https://github.com/martinrotter/rssguard\"" -DAPP_URL_ISSUES="\"https://github.com/martinrotter/rssguard/issues\"" -DAPP_URL_ISSUES_NEW="\"https://github.com/martinrotter/rssguard/issues/new\"" -DAPP_URL_WIKI="\"https://github.com/martinrotter/rssguard/wiki\"" -DAPP_USERAGENT="\"RSS Guard/3.7.1 (github.com/martinrotter/rssguard)\"" -DAPP_DONATE_URL="\"https://martinrotter.github.io/donate\"" -DAPP_SYSTEM_NAME="\"Linux\"" -DAPP_SYSTEM_VERSION="\"aarch64\"" -DAPP_REVISION="\"\"" -DQT_USE_QSTRINGBUILDER -DQT_USE_FAST_CONCATENATION -DQT_USE_FAST_OPERATOR_PLUS -DUNICODE -D_UNICODE -DUSE_WEBENGINE -DRSSGUARD_DLLSPEC=Q_DECL_EXPORT -DQT_NO_DEBUG -DQT_WEBENGINEWIDGETS_LIB -DQT_PRINTSUPPORT_LIB -DQT_WIDGETS_LIB -DQT_WEBENGINECORE_LIB -DQT_QUICK_LIB -DQT_GUI_LIB -DQT_SQL_LIB -DQT_QMLMODELS_LIB -DQT_WEBCHANNEL_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_XML_LIB -DQT_POSITIONING_LIB -DQT_CORE_LIB -I../../../src/librssguard -I. -I../../../src/librssguard -I../../../src/librssguard/gui -I../../../src/librssguard/gui/dialogs -I../../../src/librssguard/dynamic-shortcuts -Iui -isystem /usr/include/qt5/QtSolutions -isystem /usr/include/qt5 -isystem /usr/include/qt5/QtWebEngineWidgets -isystem /usr/include/qt5/QtPrintSupport -isystem /usr/include/qt5/QtWidgets -isystem /usr/include/qt5/QtWebEngineCore -isystem /usr/include/qt5/QtQuick -isystem /usr/include/qt5/QtGui -isystem /usr/include/qt5/QtSql -isystem /usr/include/qt5/QtQmlModels -isystem /usr/include/qt5/QtWebChannel -isystem /usr/include/qt5/QtQml -isystem /usr/include/qt5/QtNetwork -isystem /usr/include/qt5/QtXml -isystem /usr/include/qt5/QtPositioning -isystem /usr/include/qt5/QtCore -Imoc -I/usr/lib64/qt5/mkspecs/linux-g++ -o mimesis.o ../../../src/librssguard/3rd-party/mimesis/mimesis.cpp
../../../src/librssguard/3rd-party/mimesis/mimesis.cpp:58:3: error: narrowing conversion of '-1' from 'int' to 'char' [-Wnarrowing]
   58 |   };
      |   ^
../../../src/librssguard/3rd-party/mimesis/mimesis.cpp:58:3: error: narrowing conversion of '-1' from 'int' to 'char' [-Wnarrowing]
../../../src/librssguard/3rd-party/mimesis/mimesis.cpp:58:3: error: narrowing conversion of '-1' from 'int' to 'char' [-Wnarrowing]
../../../src/librssguard/3rd-party/mimesis/mimesis.cpp:58:3: error: narrowing conversion of '-1' from 'int' to 'char' [-Wnarrowing]
```

These lines are repeated many times after this.

#### Other information (logs, see Wiki)

Fedora Rawhide, Rssguard 3.7.1, GCC 10.2.1

Full logs for 15 days at:
https://koji.fedoraproject.org/koji/taskinfo?taskID=50384904

--- a/src/librssguard/3rd-party/mimesis/mimesis.cpp
+++ b/src/librssguard/3rd-party/mimesis/mimesis.cpp
@@ -38,7 +38,7 @@ using namespace std;
 
 namespace Mimesis {
   static const string base64 = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/";
-  static const char base64_inverse[256] = {
+  static const signed char base64_inverse[256] = {
     -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1,
     -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1,
     -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, 62, -1, -1, -1, 63,

